src/v/coproc/event_listener.cc

```diff
@@ -84,13 +103,21 @@ event_listener::event_listener(ss::sharded<pacemaker>& pacemaker)
   , _dispatcher(pacemaker, _abort_source) {}
 
 ss::future<> event_listener::start() {
-    return _client.connect().then([this] {
-        (void)ss::with_gate(_gate, [this] {
-            return ss::do_until(
-              [this] { return _abort_source.abort_requested(); },
-              [this] { return do_start(); });
-        });
+    (void)ss::with_gate(_gate, [this] {
+        return ss::do_until(
+          [this] { return _abort_source.abort_requested(); },
+          [this] {
+              return do_start().handle_exception(&err_handler).finally([this] {
+                  return ss::sleep_abortable(2s, _abort_source)
+                    .handle_exception([](std::exception_ptr) {
+                        vlog(
+                          coproclog.debug,
+                          "event_listener event loop exiting abruptly");
+                    });
```

Comment:
This is always going to be due to abort.

Suggested options:
* Catch the expected exception type and log it.
* Make the message more specific
* Don't log

Reply:
Choosing not to log